.editorconfig

```diff
@@ -116,7 +117,7 @@ ij_java_generate_final_locals = true
 ij_java_generate_final_parameters = true
 ij_java_generate_use_type_annotation_before_type = true
 ij_java_if_brace_force = never
-ij_java_imports_layout = *, |, javax.**, java.**, |, $*
+ij_java_imports_layout = $*,|,*,|,javax.**,java.**
```

Comment:
This makes it so that IntelliJ doesn't reorganize the way spotless organizes the imports.